gradle/libs.versions.toml

```diff
@@ -23,9 +23,9 @@ androidx-arch = "2.2.0"
 androidx-test-orchestrator = "1.4.2"
 androidx-sqlite = "2.3.1"
 benasher-uuid = "0.8.0"
-ktx-datetime = { strictly = "0.4.0" }
+ktx-datetime = { strictly = "0.4.1" }
 ktx-serialization = "1.5.1"
-ktx-atomicfu = "0.22.0"
```

Comment:
Doesn't look like an update.

Is the rollback to 0.20.0 intended?

Reply:
nope, reverted now

Reply:
Yes it's on purpose because both `atomicfu` 0.22 and `datetime` 0.4.1 duplicate this bin file

```
> A failure occurred while executing com.android.build.gradle.internal.tasks.MergeJavaResWorkAction
   > 2 files found with path 'META-INF/versions/9/previous-compilation-data.bin' from inputs:
```

I downgraded `atomicfu` because I recently updated it from 0.18.5

That should be fixed in 0.4.2
https://github.com/Kotlin/kotlinx-datetime/issues/304